code/controllers/subsystem/mobs.dm

```diff
@@ -32,11 +32,12 @@ SUBSYSTEM_DEF(mobs)
 	//cache for sanic speed (lists are references anyways)
 	var/list/currentrun = src.currentrun
 	var/times_fired = src.times_fired
+	var/delta_time = wait / (1 SECONDS)	// TODO: Make this actually responsive to stuff like pausing and resuming
```

Comment:
If you plan on making delta time reflect subsystems taking too long to fire you're going to have a hell of a time. Am I misunderstanding you?

Reply:
Not at all. That's exactly what I had in mind.

Reply:
It's not a current project though.